main.rb

```diff
@@ -1,11 +1,34 @@
-require './person'
-require './decorator'
-require './capitalize_decorator'
-require './trimmer_decorator'
+require_relative 'app'
 
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+def list_options
+  puts "\n"
+  puts 'Enter number to pick an option:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '7. End'
+end
+
+def main
+  app = App.new
+  loop do
+    list_options
+    option = gets.chomp.to_i
+    options = {
+      1 => -> { app.list_books },
```

Comment:
- [ ] Good job with your options, but there seems to be a little problem when trying to list all books, its showing "No people".

   ![image](https://github.com/anshuman7negi/school-library/assets/107952724/1e01e489-9416-44c9-9f4a-f2319e7909ad)